h2o-r/h2o-package/R/w2vutils.R

```diff
@@ -27,7 +27,6 @@ h2o.findSynonyms <- function(word2vec, word, count = 20) {
     res <- .h2o.__remoteSend(method="GET", "Word2VecSynonyms", model = word2vec@model_id,
                              word = word, count = count)
     fr <- data.frame(synonym = res$synonyms, score = res$scores)
-    fr[with(fr, order(score, decreasing = TRUE)),]
```

Comment:
The result is sorted on the Java side; there is no need to sort it again.

Reply:
Thanks, @valenad1, for suggesting to keep sorting on the client side. We can't guarantee the deserialization to keep the order so that I will keep it here.